stacks/spica/src/dashboard/pages/tutorial/tutorial.component.html

```diff
@@ -1,6 +1,88 @@
 <div class="content-container">
   <div class="logo">
-    <img src="assets/logo-1.png" />
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 111.1 111.1">
```

Comment:
We have a pr that fixes this bug. Can we revert this change?

Reply:
done